main.go

```diff
@@ -164,6 +167,7 @@ func main() {
 	var group run.Group
 	var ingestCollector ingest.Collector
 	var streamingConnector streaming.Connector
+	namespaceHandlers := make([]namespace.Handler, 0)
```

Comment:
```suggestion
	var namespaceHandlers []namespace.Handler
```

append will initialize the slice